def IOExceptionWithCause():
    '''public IOExceptionWithCause(final String message, final Throwable cause)
    public IOExceptionWithCause(final Throwable cause)
    '''
